sdk/search/azure-search-documents/azure/search/documents/_service/_datasources_client.py

```diff
@@ -15,7 +15,7 @@
 
 if TYPE_CHECKING:
     # pylint:disable=unused-import,ungrouped-imports
-    from ._generated.models import DataSource
+    from ._generated.models import SearchIndexerDataSource
```

Comment:
The client name should be {Model}Client here and elsewhere, so `SearchIndexerDataSourceClient`. Getters from `SearchServiceClient` should match (for now; we're discussing with archboard but this would be currently consistent).

Reply:
This is the name of the model in the generated code

Reply:
The current name of this client is `SearchDataSourcesClient` so I guess it would also need to be updated (in a different place)